### ISSUE TITLE
A11y: When Google Street View is open, focus does not move to the street view

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#399]
## Describe the task
When opening Google Street view for a single property, the focus is still on the single property sidepanel where you triggered the view from. 

Here's what the experience is like for screenreaders:

https://www.loom.com/share/605fba1977af458293337021e074ed47?sid=998d512e-f4a3-4d6f-8cf0-c08847a260c7

## Acceptance Criteria

- [ ] When opening the full screen Google street view, focus will move from the trigger button to the street view container.

Note: this issue is just for moving focus, not for fully working keyboard/screenreader use within the street view

**Comment by Amberroseweeks** [comment]
Hi, i'd like to assign myself this task if it is still available. Thank you!

**Comment by Amberroseweeks** [comment]
Just sharing an update, I have focus applied to the streetview but now that I am in here I am thinking we should add a couple more updates like:
-Close iFrame on pressing ESC key
-Have Close button receive focus first 

So I am working on having those items included. @bacitracin @marvieqa let me know if there is anything else you think should be added
